README.md

```diff
@@ -42,6 +42,20 @@ You have a list of available commands in the project manager. To run this projec
  * Debugger is active!
  * Debugger pin code: 281-707-369
 ```
+
+### Creating an admin user
+
+```shell
+$ python manage.py shell
+>>> import project.user.models as umodels
+>>> umodels.User.create('admin', 'youradminpass', True)
+<User: admin>
+>>> admin = umodels.User.objects.filter(username='admin').first()
+>>> admin.is_admin = True
+>>> admin.save()
+<User: admin>
+```
```

Comment:
This is a great idea! There is a command that allows you to create an admin user:
```shell
(flask-template-project) $ python manage.py create-user --help
usage: manage.py create-user [-?] [-u USERNAME] [-p PASSWORD] [--is_admin]

Creates user by giving username and password

optional arguments:
  -?, --help            show this help message and exit
  -u USERNAME, --username USERNAME
  -p PASSWORD, --password PASSWORD
  --is_admin
(flask-template-project) $ python manage.py create-user -u "username" -p "password" --is_admin
[INFO] - Running CreateUserCommand with arguments {'username': 'username', 'password': 'password', 'is_admin': True}
[INFO] - User "username" was successfully created!
(flask-template-project) $
```
Actually can you update that information to the above?